solution/backend/regulations/urls.py

```diff
@@ -51,8 +51,8 @@
     path('resources/', ResourcesView.as_view(), name='resources'),
     path('statutes/', StatuteView.as_view(), name='statutes'),
     path('reg_redirect/', RegulationRedirectView.as_view(), name="reg_redirect"),
-    path('policy-repository/', PolicyRepositoryView.as_view(), name='policy-repository'),
-    path('policy-repository/search/', PolicyRepositorySearchView.as_view(), name='policy-repository-search'),
+    path('policy-repository/', PolicyRepositoryView, name='policy-repository'),
```

Comment:
we still need the policy-repository URL and view to redirect people with existing links/bookmarks to the new subjects page